autopts/ptsprojects/mynewt/l2cap.py

```diff
@@ -134,6 +134,8 @@ def test_cases(ptses):
                   else "FALSE")),
               TestFunc(lambda: pts.update_pixit_param(
                   "L2CAP", "TSPX_psm_encryption_required", format(le_psm, '04x'))),
+              TestFunc(lambda: pts.update_pixit_param(
+                  "L2CAP", "TSPX_l2ca_num_concurrent_credit_based_connections", "2")),
```

Comment:
Is this a typo or is this really the PIXIT name? `l2ca`

Reply:
This is a typo, but in PTS, I guess we need to propagate it for now :)

Reply:
Ok, this is a typo in pts.